src/pactflow/client/prompt-utils.ts

```diff
@@ -1,5 +1,5 @@
 import type { Server } from "@modelcontextprotocol/sdk/server/index.js";
-import type { GetInputFunction } from "../../common/types.js";
+import { type GetInputFunction, ToolError } from "../../common/types.js";
```

Comment:
should type be outside the braces?

Reply:
you can use it either way - inline type imports were added in TypeScript 4 (3.9?) it allows you to import a type alongside actual code on one line. (we're importing `ToolError` not as a type)

Reply:
ah fair enough, i clearly dont know enough about typescript 😅